python/tests/complex_type_tests/test_arr_of_row.py

```diff
@@ -79,3 +79,28 @@ def __init__(self):
                       c1_arr[1][2] AS c12_elmnt,
                       c2_arr[2][1] AS c21_elmnt
                       FROM arr_of_row_tbl"""
+
+
+class cmpxtst_arr_of_row_arr_idx_outbound(TstView):
+    def __init__(self):
+        # checked manually
+        self.data = [
+            {"id": 0, "c15_val": None, "c151_elmnt": None},
+            {"id": 1, "c15_val": None, "c151_elmnt": None},
+        ]
+        self.sql = """CREATE MATERIALIZED VIEW arr_of_row_arr_idx_outbound AS SELECT
+                      id,
+                      c1_arr[5] AS c15_val,
+                      c1_arr[5].i1 AS c151_elmnt
+                      FROM arr_of_row_tbl"""
+
+
+# ignore => error: Unknown field 'i5' when inner row column does not exist
```

Comment:
we are not really writing negative tests now, can you replace the tests that produce compilation errors with tests that run?
we should test compilation errors too, but probably in a different way

Reply:
rather, what you should test is that you can access a legal field in an element of the array that does not exist, and then the result is NULL